### PR TITLE
Sveta_Assignment_23

### PR DESCRIPTION
**Pull request by svetakanani**
### Evidence :file_folder:

image

### Checklist before requesting a review :ballot_box_with_check:

- [x] I have performed a self-review of my code
- [x] My changes generate no compile time errors

### CODE REVIEW

File: src/sveta/Assignment_23_CharacterFriquency.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+public class Assignment_23_CharacterFriquency {
+
+	public static void frequencyOfCharacter() {
+		String str="technocredits";
+		
+		char ch=' ';
+		int frequency = 0;
+		for(int i=0;i<=str.length()-1;i++) {
+			ch =  str.charAt(i);
+            if(ch == str.charAt(i)) {
```
Comment (MargiVegada) [inline src/sveta/Assignment_23_CharacterFriquency.java:19]: Improper alignment

File: src/sveta/Assignment_23_CharacterFriquency.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+                ++frequency;
+            }
+            
+        }
+		System.out.println("Frequency of " + ch + " = " + frequency);
+
```
Comment (MargiVegada) [inline src/sveta/Assignment_23_CharacterFriquency.java:25]: Don't put unnecessary blank lines

**Review by MargiVegada — APPROVED** [review]
Need to work on indentation

File: src/sveta/Assignment_23_CharacterFriquency.java (hunk 1 above)
Comment (DharmaKanani) [reply src/sveta/Assignment_23_CharacterFriquency.java:19]: i can't see code like above comment in file. how i can solve. can you once check my screenshot attached with this pr.my code look like above mention screenshot.

**Pull request merged**